### PR TITLE
Added changes required in helm charts for Litmus 3.0.0

### PR DESCRIPTION
**Pull request by Saranya-jena**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] DCO signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

**Review by Jonsy13 — APPROVED** [review]

File: charts/litmus/values.yaml [hunk 1]
```diff
@@ -98,15 +98,15 @@ portal:
     ##       maxSurge: 1
     ##       maxUnavailable: 25%
     automountServiceAccountToken: false
-    securityContext:
-      runAsUser: 2000
-      allowPrivilegeEscalation: false
-      runAsNonRoot: true
+    # securityContext:
```
Comment (imrajdas) [inline charts/litmus/values.yaml:101]: why did we remove the securityContext?

Comment (imrajdas) [reply charts/litmus/values.yaml:101]: add fsGroup instead

Comment (Jonsy13) [reply charts/litmus/values.yaml:101]: This is commented for now, Need to make it compatible with redhat images/users. 
Will be added back in a separate PR later.

**Review by imrajdas — APPROVED** [review]

File: charts/litmus/values.yaml (hunk 1 above)
Comment (imrajdas) [reply charts/litmus/values.yaml:101]: ok

**Review by imrajdas — APPROVED** [review]

**Pull request merged**